packages/front-end/pages/settings/billing.tsx

```diff
@@ -74,6 +74,19 @@ const BillingPage: FC = () => {
     );
   }
 
+  if (organization.isVercelIntegration) {
```

Comment:
do we need to worry about the (eventual) intersection of `accountPlan === "enterprise"` and `organization.isVercelIntegration`?